R/input-switch.R

```diff
@@ -45,6 +45,7 @@
 input_switch <- function(id, label, value = FALSE, width = NULL) {
   tag <- input_checkbox(id, label, class = "form-check form-switch", value = value, width = width)
   tag <- tag_require(tag, version = 5, caller = "input_switch()")
+  tag <- tagList(tag, toggle_input_binary_js())
```

Comment:
At a minimum this should be a singleton, but it also feels like we should bring this in via a `htmlDependency()` (mainly so that the JS lives in one place, which makes py-shiny's job easier?)

```suggestion
  tag <- tagList(tag, singleton(toggle_input_binary_js()))
```

Reply:
I had that originally and then wondered about race conditions and took it out. Happy to move it to a dependency.